### ISSUE TITLE
I have weights for every example. Need to multiple them with the loss function

### ISSUE THREAD

**Issue by bjayakumar** [google-research/bert#208]
How can I do it correctly? 

Example

Sentence 1, wgt1, label1
Sentence 2, wgt2, label2
  ...

I need multiple this weight with loss function and train them.

Please help me.

**Comment by shuaihhx** [comment]
You may add the weights into features when building examples. Then you can use it in function model_fn_builder and then in function create_model.
